Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM centos:centos7
+
+EXPOSE 8080
+
+ADD sendEmail-epel-7.repo /etc/yum.repos.d/
```

Comment:
I won't be able to use packages from EPEL repositories, so if you need any package from EPEL, please reach me asap :-)